src/stylus/fablevision/breed-button.styl

```diff
@@ -0,0 +1,27 @@
+.geniblocks.breed-button
+  background: url('../resources/fablevision/layout-c/breed_surround.png')
+  width: 370px
+  height: 369px
+  display: flex
+  justify-content: center
+  align-items: center
+
+  .clickable-button
+    width: 139px
+    height: 138px
+    display: flex
+    justify-content: center
+    align-items: center
+    background: url('../resources/fablevision/layout-c/breed_button_UNCLICKED.png')
+    cursor: pointer
+
+    &:active
+    &.disabled
+      cursor: default
+      background: url('../resources/fablevision/layout-c/breed_button_CLICKED.png')
+
+    .text
```

Comment:
`text` implies that it's applicable to all text. I would suggest something like `button-label` which specifies the `font-family`, since all buttons are likely to use the same font, and then another class for `breed-button-label` which specifies the `font-size`, since that's likely to be unique to the breed button.